sway-lib-core/src/codec.sw

```diff
@@ -335,6 +335,54 @@ where
     }
 }
 
+// Tuples
+
+impl<T> AbiEncode for (T, T)
```

Comment:
Doesn't tuples need to be generic in each argument?

```sway
impl<A, B> AbiEncode for (A, B)
where
    A: AbiEncode,
    B: AbiEncode,
```

Reply:
Ah, good catch! Will implement.